Sail/src/Sail/entities/systems/Gameplay/ProjectileSystem.cpp

```diff
@@ -34,17 +33,14 @@ void ProjectileSystem::update(float dt) {
 		auto projComp = e->getComponent<ProjectileComponent>();
 		for (auto& collision : projectileCollisions) {
 			// Check if a decal should be created
-			if (projComp->timeSinceLastDecal > m_splashMinTime &&
-				glm::length(e->getComponent<MovementComponent>()->oldVelocity) > 0.7f) {
+			if (glm::length(e->getComponent<MovementComponent>()->oldVelocity) > 0.7f) {
 				// TODO: Replace with some "layer-id" check rather than doing a string check
 				if (collision.entity->getName().substr(0U, 4U) == "Map_") {
+					// Calculate rotation matrix used when placing a decal at the intersection
+					//glm::mat4 rotMat = glm::rotate(glm::identity<glm::mat4>(), Utils::fastrand() * 3.14f, glm::vec3(0.0f, 0.0f, 1.0f));
```

Comment:
Do you need to save this code?

Reply:
It is kept as we might need that line when or if decals are added again in the future, thats why I put that comment there as well